lib/broccoli/angular-broccoli-less.js

```diff
@@ -0,0 +1,69 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+try {
+  let less;
+
+  if (process.platform === 'win32') {
+    require.resolve(`${process.env.PWD}/node_modules/less`);
```

Comment:
Try if the npm module exists in the project and fails if not. Thats why the whole plugins are in `try { } catch (e) { }` blocks.